src/frequenz/sdk/microgrid/__init__.py

```diff
@@ -137,6 +137,93 @@
 [`propose_power`][frequenz.sdk.timeseries.ev_charger_pool.EVChargerPool.propose_power],
 which accepts values in the {{glossary("psc", "Passive Sign Convention")}} and supports
 only charging.
+
+# Component pools
+
+The SDK provides a unified interface for interacting with sets of Batteries, EV
+chargers and PV arrays, through their corresponding `Pool`s.
+
+| pool type     | constructor                                                         |
+|---------------|---------------------------------------------------------------------|
+| BatteryPool   | [microgrid.battery_pool][frequenz.sdk.microgrid.battery_pool]       |
+| EVChargerPool | [microgrid.ev_charger_pool][frequenz.sdk.microgrid.ev_charger_pool] |
+| PVPool        | [microgrid.pv_pool][frequenz.sdk.microgrid.pv_pool]                 |
```

Comment:
Alternatively (I'm not sure the table format here provides much extra if it is only a name and a link basically):

```suggestion
* [Battery pool][frequenz.sdk.microgrid.battery_pool]
* [EV charger pool][frequenz.sdk.microgrid.ev_charger_pool]
* [PV pool][frequenz.sdk.microgrid.pv_pool]
```

Or if you prefer a more code-oriented version:


```suggestion
* [`battery_pool`][frequenz.sdk.microgrid.battery_pool]
* [`ev_charger_pool`][frequenz.sdk.microgrid.ev_charger_pool]
* [`pv_pool`][[frequenz.sdk.microgrid.pv_pool]
```